src/test/java/com/ericsson/ei/test/utils/TestConfigs.java

```diff
@@ -46,42 +47,26 @@ public static synchronized void init() throws Exception {
     }
 
     private static synchronized void setUpMessageBus() throws Exception {
-        LOGGER.debug("Debug:setting up message buss");
+        LOGGER.debug("Debug:setting up message bus");
```

Comment:
Why the extra debug inside the log message?

Reply:
Just to debug....well, now it is removed